code-challenges.js

```diff
@@ -12,38 +12,85 @@
 const fruit1 = "apple"
 const fruit2 = "banana"
 // Expected outcome: "banana"
-
-const fruit3 = "cherry"
-const fruit4 = "kiwi"
+// const longerString = (str1, str2) => {
+//     if ( str1.length > str2.length) {
+//     return str1;
+//     } else if (str2.length > str1.length) {
+//     return str2;
+//     } else {
+//         return "Something went wrong"
+//     }
+// };
+//     console.log(longerString(fruit1,fruit2))
+
+
+// const fruit3 = "cherry"
+// const fruit4 = "kiwi"
 // Expected outcome: "cherry"
 
 // Explain your code:
 
 // --------------------1) Create a function that takes in an array of numbers and returns the sum of all the numbers. Use the test variables provided below.
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
-// Expected output: 15
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+
+sumOfAllNumbers = (numbers) => {
+    let total = 0;
+    for (let i = 0; i < numbers.length; i++) {
+        total += numbers[i]
+    }
+    return total
+}
+
+console.log(sumOfAllNumbers(padres1984WorldSeriesRuns)); //15
+console.log(sumOfAllNumbers(padres1998WorldSeriesRuns)); //17
```

Comment:
you forgot the function declaration  `const` in your fxn definition.  Otherwise, looks great!